src/commands/utility/emojilist.js

```diff
@@ -0,0 +1,34 @@
+const { Command, SwitchbladeEmbed, PaginatedEmbed, CommandError } = require('../../')
+
+module.exports = class EmojiList extends Command {
+  constructor (client) {
+    super({
+      name: 'emojilist',
+      aliases: ['emojis', 'emojislist', 'guildemojis', 'serveremojis'],
+      category: 'utility',
+      requirements: { guildOnly: true, botPermissions: ['ADD_REACTIONS', 'EMBED_LINKS'] }
+    }, client)
+  }
+
+  async run ({ message, t, author, channel }) {
+    const pages = message.guild.emojis.cache
+      .filter(emoji => emoji.available)
+      .reduce((descriptionArr, emoji, id) => {
```

Comment:
Stop using `reduce`, it's very confusing. You can do same with:

```js
const pages = [];
const emojis = message.guild.emojis.cache.filter(emoji => emoji.available)

emojis.forEach((emoji) => {
  const line = `${emoji.toString()} **${emoji.name}** \`\`<:${emoji.name}:${id}>\`\``

  const [lastPage] = pages.slice(-1)

  if (lastPage && [...lastPage, line].join('\n').length < 2048) lastPage.push(line)
  else pages.push([line]) 
});
```

Reply:
Not applying this change because it was said to not do it while having an internal talk with the team.